Screenbox/Pages/PlayerPage.xaml.cs

```diff
@@ -164,6 +163,19 @@ private void CoreTitleBar_LayoutMetricsChanged(CoreApplicationViewTitleBar sende
             RightPaddingColumn.Width = new GridLength(Math.Max(sender.SystemOverlayLeftInset, sender.SystemOverlayRightInset));
         }
 
+        private void AccessKeyManager_OnIsDisplayModeEnabledChanged(object sender, object args)
+        {
+            ViewModel.KeyTipsVisible = AccessKeyManager.IsDisplayModeEnabled;
+            if (AccessKeyManager.IsDisplayModeEnabled)
+            {
+                ViewModel.ControlsHidden = false;
+            }
+            else
+            {
+                ViewModel.TryHideControls();
```

Comment:
A better approach is to use `DelayHideControls()`, though we'd have to expose it as public.